src/views/Player.vue

```diff
@@ -3,56 +3,50 @@
     <v-row>
       <v-col cols="12">
         <v-card tile>
-          <v-card-title class="justify-space-between">
-            <div
-              style="display: flex; flex-direction: row; align-items: center"
-            >
-              <span>
-                {{ $t("views_player.profile") }} {{ profile.battleTag }}
-              </span>
-              <div v-if="aliasName">({{ aliasName }})</div>
-              <div
-                style="display: flex; flex-direction: row; margin-left: 25px"
-              >
-                <SeasonBadge
-                  v-for="season in seasonsWithoutCurrentOne"
-                  :season="season"
-                  :key="season.id"
-                  :on-click="selectSeason"
-                />
-              </div>
-            </div>
-            <div>
-              <gateway-select @gatewayChanged="gatewayChanged" />
-              <v-menu offset-x v-if="!!seasons && seasons.length > 0">
-                <template v-slot:activator="{ on }">
-                  <v-btn tile v-on="on" class="ma-2 transparent">
+          <v-card-title>
+            <v-row no-gutters>
+              <v-col :align-self="'center'">
+                <span>{{ $t("views_player.profile") }} {{ profile.battleTag }}</span>
+                <span v-if="aliasName" class="ml-1">({{ aliasName }})</span>
+                <span class="mr-2" /> <!-- add some space between name and season badges -->
+                <span v-for="season in seasonsWithoutCurrentOne" :key="season.id" class="ml-1">
+                  <SeasonBadge :season="season" :on-click="selectSeason" :size="'sm'" />
```

Comment:
Thanks for the contribution, my only request is to make the `:size` dependent on device size, by hardcoding `sm`, it looks teeny on desktop scale:
![image](https://user-images.githubusercontent.com/40397054/140607990-8b40b85b-e9fe-4961-b00f-161838031ab7.png)

Reply:
The smaller (sm) version was indented for all screen sizes. I think it looks more convienient this way. But sure, I can change that and replace the `size` property with media queries.

Reply:
I played a little around and reverted back to the "old" 24px size for all device sizes. It looks decent imo, @Cepheid-UK what do you say?

<img width="1373" alt="Bildschirmfoto 2021-11-07 um 01 46 36" src="https://user-images.githubusercontent.com/8781699/140628196-d2d5a8ea-3947-4361-bacd-0c500bd2a752.png">
<img width="429" alt="Bildschirmfoto 2021-11-07 um 01 47 14" src="https://user-images.githubusercontent.com/8781699/140628194-80310a22-bfda-4fb1-9845-281bf6fb74f4.png">
<img width="325" alt="Bildschirmfoto 2021-11-07 um 01 47 27" src="https://user-images.githubusercontent.com/8781699/140628193-729d72f7-e7d8-426c-8aff-16437974f5b0.png">